conf/config.properties

```diff
@@ -255,6 +255,10 @@ classification.language.spanish=que,de,no,a,la,el,es,y,en,lo,un,por,qu
 classification.language.dutch=de,van,een,het,en,in,is,dat,op,te,De,zijn,voor,met,die,niet,aan,er,om,Het,ook,als,dan,maar,bij,of,uit,nog,worden,door,naar,heeft,tot,ze,wordt,over,hij,In,meer,jaar,was,ik,kan,je,zich,al,hebben,geen,hun,we,wat,Een,Maar,werd,moet,wel,kunnen,Dat,nu,dit,deze,zal,Ik,veel,zo,En,andere,nieuwe,zou,twee,moeten,onder,eerste,haar,Van,wil,tegen,men,mensen,gaat,tussen,grote,waar,goed,maken,dus,alleen,Hij,Op,frank,ons,u,daar,na,had,gaan,alle,Als,Er,��n
 
 # flags
+flag.log.dao = true;
+flag.severe.dao = true;
+flag.debug.dao = true;
+flag.trace.dao = true;
```

Comment:
@singhpratyush , the log levels that are managed in `log4j2.properties` and `log-to-file.properties` files logs `INFO` log levels. This means all logs generated by `org.eclipse.jetty.util.log.Log` are logged. these logs are all INFO according to `log4j2.properties` and `log-to-file.properties`. And these log levels (trace, info, warn, debug) dealt here are logged as `INFO` in the loklak.log file are of different log levels according to `org.eclipse.jetty.util.log.Log`.